tests/datasets/test_winer.py

```diff
@@ -0,0 +1,92 @@
+import pytest
+import os
+
+from wellcomeml.datasets.winer import create_train_test, _load_data_spacy
+
+@pytest.fixture(scope="module")
+def define_paths():
+
+    NE_path = 'tests/test_data/mock_winer_CoarseNE.tar.bz2'
+    docs_path ='tests/test_data/mock_winer_Documents.tar.bz2'
+    vocab_path = 'tests/test_data/mock_winer_document.vocab'
+
+    # These will be generated and then deleted in these tests
+    train_processed_path = 'tests/test_data/temp_train_sample.txt'
+    test_processed_path = 'tests/test_data/temp_test_sample.txt'
+
+    return (
+        NE_path, docs_path, vocab_path,
+        train_processed_path, test_processed_path
+        )
+
+def test_create_train_test(define_paths):
+
+    (NE_path, docs_path, vocab_path,
+        train_processed_path, test_processed_path) = define_paths
+    # Create the train/test data
+    n_sample = 10
+    prop_train = 0.5
+
+    create_train_test(
+        NE_path, vocab_path, docs_path,
+        train_processed_path, test_processed_path,
+        n_sample, prop_train, rand_seed=42
+        )
+    assert os.path.exists(train_processed_path) and os.path.exists(test_processed_path)
```

Comment:
possibly a better test is to test a function that creates and returns the datasets before saving and test some elements. i think it is generally suggested to avoid IO and network operations in tests unless you are explicitly testing for those i.e. a function whose purpose is solely to do IO.

Reply:
i will just delete this test then. the next test does as you say anyway - creates the datasets and tests some elements of them